fundamentals-recap/math-primer.md

```diff
@@ -0,0 +1,170 @@
+# Math Primer
+
+## Goal
+
+Problem-solving in programming is almost always centered on _data_. Often times, it's useful to be aware of the following math concepts. Knowing the definition of these concepts can become tools you remember to use in the future.
+
+The goal of this document is to be a resource. We recommend spending your energy on this document reading through it and taking notes on it once, and then reviewing briefly every few months.
+
+This document covers the following:
+
+Thinking about numbers:
+
+1. Decimal Number System
+1. Using _n_ as a Variable in Math
+1. Coordinate Planes
+
+Working with collections of numbers:
+
+1. Maximum and Minimum
+1. Sorting
+1. Average or Mean
+1. Floor and Ceiling Functions
+
+## Decimal Number System
+
+The _decimal number system_ or _base-ten number system_ is the numbering system most humans use in everyday life.
+
+This numbering system includes - ten unique symbols known as digits: 0, 1, 2, 3, 4, 5, 6, 7, 8, 9 - the decimal point (floating point/dot) symbol to represent decimal fractions, such as 2.45
+
+### !callout-info
+
+## Origin of Base 10
+
+"Legend says" that most base-ten systems possibly originate from counting on ten fingers.
+
+### !end-callout
+
+## Using _`n`_ as a Variable in Math
```

Comment:
To me, this section really blurs the line between variables as a mathematical concept, and variables as a programming concept. For the brevity of the material, that may be fine, but if this is meant to help students be more comfortable with mathematical terminology and expressions (which often come up in computing), then keeping thing focused on variables in equations, and looking at simple algebraic operations might be more useful.

What really is the expected level of mathematics for incoming students? I do occasionally encounter students who don't seem very comfortable expressing mathematical relationships between quantities with equations. Having a good grounding in math is very useful for any programmer. While the classroom curriculum is too short for a full math primer, it would be great to provide some links to external sources that students could use to fill in the gaps, and if there are core expectations, I'd like to see the Ada curriculum include that information, at least in a reference form.

Reply:
Good point.

I'll keep thinking on this; I'm working on determining the right data for what math is expected at Ada. 😄  I'm mostly trying to convert [this document](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/04-cs-fundamentals/classroom/Essential%20Mathematics.md) for ease of transition from C14-C15!